### ISSUE TITLE
Passport crashes server on login

### ISSUE THREAD

**Issue by dajinchu** [ryan-bradford/WHAM#4]
When I try to log in with incorrect credentials, it redirects me back to the login page, as expected.
However, when I try to log in with _correct_ credentials, the server throws an error "secretOrPrivateKey must have a value"

I think the fix is to add to index.js:
`app.use(session({ secret: process.env.SECRET}));`

and of course add a secret string in .env

I'm just not sure if the problem is only on my end. @ryan-bradford can you reproduce this issue?

**Comment by ryan-bradford** [comment]
Ahhhh I'm sorry I think this is my fault. When I set up the JSON Web Tokens, it requires a secret key to encrypt the JWT. I added it to my .env (called JWT_SECRET) but I forgot to tell you. So on my end, when I login it returns (in plain text) the JWT as the next page, but on your end it crashes the site because JWT_SECRET does not exist. My bad....

Also I love the temporary WHAM logo hahahahaha.
